src/go/k8s/controllers/redpanda/metric_controller.go

```diff
@@ -15,13 +15,13 @@ import (
 var (
 	redpandaClusters = prometheus.NewGauge(
 		prometheus.GaugeOpts{
-			Name: "redpanda_cluster_total",
-			Help: "Number of Redpanda cluster running",
+			Name: "redpanda_clusters_total",
+			Help: "Number of Redpanda clusters running",
 		},
 	)
 	desireRedpandaNodes = prometheus.NewGaugeVec(
 		prometheus.GaugeOpts{
-			Name: "desire_redpanda_nodes_total",
+			Name: "desired_redpanda_nodes_total",
 			Help: "Number of desire Redpanda nodes",
```

Comment:
s/desire/desired/
What do we mean by "desired"? Can we replace it with something more descriptive?
As is, it sounds to me like the number of nodes that I want to create.

Reply:
yeah, that's basically it. I think this wording is pretty common in k8s world, you have desired state and actual state, desired is how many nodes you want in this example